configuration/pih/petl/jobs/refresh-hiv-data.yml

```diff
@@ -34,6 +34,7 @@ configuration:
           - tableName: "hiv_visit"
           - tableName: "ovc_program_encounters"
           - tableName: "hivmigration_data_warnings"
+          - tableName: "pmtct_infant_delivery"
```

Comment:
Just confirming - this is intended to run against the HIV system, and not the MCH system.  If so, this is the right file for this.  The order of the tableNames might be something you want to keep an eye on - keep them alphabetical or organized by data.  Putting this after the data warning table feels less well organized.